support-frontend/assets/helpers/globalsAndSwitches/settings.ts

```diff
@@ -8,7 +8,7 @@ export type Status = 'On' | 'Off';
 
 export type SwitchObject = Record<string, Status>;
 
-export type Switches = {
+export type Switches = Record<string, SwitchObject> & {
```

Comment:
This was an interesting one, the original type had `experiments` followed then by `[key: string]`, given `experiments` is just a string ts was complaining that it wasn't a `SwitchObject`